lib/get_user_move.ex

```diff
@@ -0,0 +1,60 @@
+import WinningCombo
+
+defmodule GetUserMove do
+  def getMove(board, token) do
+    raw_input = _slugInput("Your Turn! Please choose an empty game space. ")
+    checked_input = _inputChecker(
+            _createValidInputArray(board, token, []), raw_input)
+
+    _replacementFunction(board, token, checked_input, [])
+
+  end
+
+  def _slugInput(messagePrompt) do
+    input = IO.gets(messagePrompt)
+    if input == :eof do
+      raise "Why wont this work"
+    else
+      String.trim(input)
+    end
+  end
+
+  def _createValidInputArray([head|tail], token, recursiveList) do
+    if elem(head, 1) == token or elem(head, 1) == "X" do
+      _createValidInputArray(tail, token, recursiveList)
+
+    else
+      _createValidInputArray(tail, token, [elem(head, 1)| recursiveList])
+    end
+
+  end
+
+  def _createValidInputArray([], token, recursiveList) do
+    recursiveList
+  end
+
+  def _inputChecker(array, testElement) do
+    if Enum.member?(array, testElement) do
+      testElement
+
+    else
+      _inputChecker(array, _slugInput("Invalid Input: Please choose an empty game space. "))
+    end
+  end
+
+  def _replacementFunction([head|tail], token, replacement, recursiveList) do
```

Comment:
Same comment on this function. I think that `replacement` is the spot getting replaced, maybe name it `spot`? And `recursiveList` seems like the resulting board. Also, to me `recursiveList` looks a lot like an accumulator for a call to the `reduce` function. Do you know about reduce yet? https://hexdocs.pm/elixir/1.12/Enum.html#reduce/2

Reply:
replacement has been changed to selected_spot